BUILDING

```diff
@@ -39,7 +39,19 @@ get the TinyDTLS code into your project, so the TinyDTLS library can be used.
  git submodule init
  git submodule update
 
-General Building (not for LwIP or Contiki - see below)
+General Building with cmake for linux/windows/macos/android (not for LwIP or Contiki - see below)
+================
+
+ cmake -E remove_directory build
+ cmake -E make_directory build
+ cd build
+ cmake .. -DENABLE_EXAMPLES=ON
+ cmake --build .
+ [sudo] cmake --build . -- install
+ to see possible options (TLS lib, doc, tests, examples etc.):
+ cmake -LH .
+
+General Building with autoconf (not for LwIP or Contiki - see below)
```

Comment:
For the Windows section right at the end of BUILDING, it should refer to Visual Studio builds as well.
````
[Previous]
You may need to edit win32\libcoap.props to update the OpenSSLRootDir and
OpenSSLRootDirDbg variables to point to the top level directory where OpenSSL
is installed so that the include, lib etc. directories are correctly set up.
Note: Make sure that you include a trailing \ in the variable definitions.
[Addition]
Alternatively you can build everything in Visual Studio with CMake.